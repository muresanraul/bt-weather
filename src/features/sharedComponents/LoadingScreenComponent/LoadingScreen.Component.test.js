import React from 'react';
import renderer from 'react-test-renderer';

import { default as LoadingScreenComponent } from './LoadingScreen.Component';

describe('LoadingScreenComponent:', () => {

  it('Component - renders correctly', () => {
    const loadingScreenComponent = renderer.create(
      <LoadingScreenComponent/>
    ).toJSON();
    expect(loadingScreenComponent).toMatchSnapshot();
  });

});
