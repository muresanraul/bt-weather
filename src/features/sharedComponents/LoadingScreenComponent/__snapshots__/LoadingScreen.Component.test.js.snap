// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreenComponent: Component - renders correctly 1`] = `
<div
  className="LoadingScreen"
>
  <div
    className="LoadingScreen--background"
  >
    <div
      className="LoadingScreen--message"
    >
      <div
        className="lds-css ng-scope"
      >
        <div
          className="lds-spin"
          style={
            Object {
              "height": "100%",
              "width": "100%",
            }
          }
        >
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
